features/core-download.feature

```diff
@@ -95,3 +95,87 @@ Feature: Download WordPress
       """
       File removed: wp-content
       """
+
+  Scenario: Installing trunk
```

Comment:
What's the expected behaviour here? Currently it just ignores the locale setting as they aren't any language specific nightly builds.